torch/linalg/__init__.py

```diff
@@ -1212,19 +1267,24 @@
 """)
 
 cond = _add_docstr(_linalg.linalg_cond, r"""
-linalg.cond(input, p=None, *, out=None) -> Tensor
+linalg.cond(A, p=None, *, out=None) -> Tensor
 
-Computes the condition number of a matrix with respect to a norm. Supports batched inputs.
+Computes the condition number of a matrix with respect to a norm.
```

Comment:
can we be more precise about the norm? "a matrix norm"?

Reply:
Good catch. Changed.